src/Search.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+function Search(){
+    const handleMouseOver = (event) =>{
```

Comment:
The handler should be named handleChange because it is triggered by the onChange event, which corresponds to changes in the input field's value. Using a descriptive name improves code readability and aligns with the actual behavior of the handler. If this component is meant to implement a search, the handler should include logic to perform the search based on the input.

Reply:
done